src/io/forsta/securesms/database/model/ThreadRecord.java

```diff
@@ -91,7 +91,11 @@ public SpannableString getDisplayBody() {
     String body = getBody().getBody();
     try {
       ForstaMessage forstaBody = getForstaMessageBody();
-      body = forstaBody.getTextBody();
+      if (forstaBody.getVote() > 0) {
+        body = "Up Vote";
```

Comment:
if vote > 0, up vote, but if not do something different?  In theory, downvotes could exist.  This seems ambiguous.